app/models/concerns/foreman_remote_execution/host_extensions.rb

```diff
@@ -11,9 +11,31 @@ module HostExtensions
       has_many :targeting_hosts, :dependent => :destroy, :foreign_key => 'host_id'
       has_many :template_invocations, :dependent => :destroy, :foreign_key => 'host_id'
       has_one :execution_status_object, :class_name => 'HostStatus::ExecutionStatus', :foreign_key => 'host_id'
+      has_many :run_host_job_tasks, :through => :template_invocations
+
+      scoped_search :relation => :run_host_job_tasks, :on => :result, :rename => 'job_task.result',
+                    :ext_method => :search_by_job_invocation,
+                    :only_explicit => true,
+                    :complete_value => { :error => 'error', :success => 'success',
+                                         :pending => 'pending', :warning => 'warning',
```

Comment:
Error/warning would be better to be hidden behind 'failed' as that's how we should the results in hosts list (there is no error or warning word there)